### ISSUE TITLE
Draggable separators can be moved to hard-to-reach locations on mobile

### ISSUE THREAD

**Issue by weirdbeardgame** [decompme/decomp.me#1202]
**Describe the bug**
On the mobile version of the site, the draggable elements can at times get stuck at the bottom of the screen, or are otherwise difficult to touch and drag.

The ASM windows at the top do not move left and right and thus I can't see the original vs the currently interpreted ASM

**To Reproduce**
Drag source code block to bottom of screen on the mobile android or iPhone rendering of the site.
See it is difficult to move.

Try to move the ASM windows left or right, see elements don't move.

**Expected behavior**

All Elements should be easy to move 

**Screenshots**

**Smartphone (please complete the following information if applicable):**
Pixel 7 using FireFox

**Additional context**
Any smaller screen mobile device will especially be difficult
